projects/aca-shared/src/lib/services/aca-mobile-app-switcher.service.ts

```diff
@@ -0,0 +1,41 @@
+import { AppConfigService } from '@alfresco/adf-core';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AcaMobileAppSwitcherService {
+  constructor(private config: AppConfigService) {}
+
+  checkForMobileApp() {
+    const mobileAppSwitchConfig: any = this.config.get<any>('mobileAppSwitch', {});
+    const ua = navigator.userAgent.toLowerCase();
+    const isAndroid = ua.indexOf('android') > -1;
+    const isIphone = ua.indexOf('iphone') > -1;
+    if (isIphone === true) {
+      const url = window.location.href;
```

Comment:
you can move this outside the "if" as it's the same for "else" too

Reply:
noted and resolved